test/ConsolePlayerTest.java

```diff
@@ -0,0 +1,11 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConsolePlayerTest {
+    @Test
+    public void createComputerPlayerWithAThrow() {
+        ConsolePlayer consolePlayer = new ConsolePlayer(Throw.ROCK);
+        assertEquals(Throw.ROCK, consolePlayer.getThrow());
```

Comment:
I was expecting ConsolePlayer to be responsible for actually getting the throw from the console instead of it just holding it. Perhaps this is just a first step towards that, but I'm not sure there will be much value in this test (just passing in the throw on the constructor and making sure that you get it back with getThrow. Also, the Player interface is empty, seems like getThrow should be declared in there so that there's a reason for having the interface.